gems/sorbet/lib/hidden-definition-finder.rb

```diff
@@ -191,8 +191,8 @@ def serialize_constants(source, rbi, klass, is_singleton, source_symbols, rbi_sy
     ret = []
 
     rbi.each do |rbi_entry|
-      # skip synthetic constants
-      next if rbi_entry["name"]["kind"] == "UNIQUE"
+      # skip duplicated constant fields
+      next if rbi_entry["name"]["kind"] == "UNIQUE" and rbi_entry["name"]["unique"] == "MANGLE_RENAME"
```

Comment:
Thanks! I didn't realize that it was mangle rename for both static fields and methods. Given that, i'm even more on board with this change.